frontend/package.json

```diff
@@ -20,64 +20,69 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "color": "^3.1.2",
-    "core-js": "^3.1.4",
+    "core-js": "^3.6.4",
+    "eslint-import-resolver-node": "^0.3.3",
+    "history": "^4.10.1",
     "i18next": "10.5.0",
     "js-cookie": "^2.2.0",
     "lodash": "^4.17.11",
     "prop-types": "^15.7.2",
     "query-string": "^6.8.1",
     "radium": "^0.25.2",
-    "react": "16.7.0",
+    "react": "^16.13.1",
```

Comment:
Okay, let's talk about it and we can revert to the old react if necessary and decide to upgrade later :-)

Reply:
I would say to keep it since, from my first tests, I couldn't reproduce the error.

If the error reappear we most likely keep the old react version.

If there isn't the error, that'll be great, we'll can keep being up to date with react version